src/views/consent/ConsentBeacon.js

```diff
@@ -31,15 +31,20 @@ define(['okta'], function (Okta) {
       </div>\
       <div class="logo-wrapper consent-beacon-client">\
         <div class="consent-beacon-border"/>\
-          {{#if clientLogo}}\
-            <img class="client-logo" src="{{clientLogo}}" />\
+          {{#if customLogo}}\
+            <img class="client-logo custom-logo" src="{{customLogo}}" />\
+          {{else if defaultLogo}}\
```

Comment:
this will never be empty right? Probably this should be an ```{{else}}``` only, without any other condition. That way we know we will have a logo no matter what.

Reply:
I felt this was safer, is baseUrl always available? If yes, I can put back to just else.

Reply:
even if ```baseUrl``` is not, you are returning a string that is not empty, so default logo will always be there.
And well, ```baseUrl``` is [always available](https://github.com/okta/okta-signin-widget#new-oktasigninconfig), it is the only required config param to bootstrap the widget.

Reply:
Good point, thank you